src/server/auth/cmds/cmds.go

```diff
@@ -0,0 +1,82 @@
+package cmds
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"strings"
+	"time"
+
+	"golang.org/x/net/context"
+
+	"google.golang.org/grpc"
+
+	"github.com/pachyderm/pachyderm/src/client"
+	"github.com/pachyderm/pachyderm/src/client/auth"
+	"github.com/pachyderm/pachyderm/src/client/pkg/config"
+	"github.com/pachyderm/pachyderm/src/server/pkg/cmdutil"
+
+	"github.com/spf13/cobra"
+)
+
+var githubAuthLink = `http://github.com/login/oauth/authorize?client_id=d3481e92b4f09ea74ff8&redirect_uri=https%3A%2F%2Fpachyderm.io%2Flogin-hook%2Fdisplay-token.html`
+
+// LoginCmd returns a cobra.Command to login to a Pachyderm cluster with your
+// GitHub account. Any resources that have been restricted to the email address
+// registered with your GitHub account will subsequently be accessible.
+func LoginCmd() *cobra.Command {
+	login := &cobra.Command{
+		Use:   "login",
+		Short: "Login to Pachyderm with your GitHub account",
+		Long: "Login to Pachyderm with your GitHub account. Any resources that " +
+			"have been restricted to the email address registered with your GitHub " +
+			"account will subsequently be accessible.",
+		Run: cmdutil.Run(func([]string) error {
+			// TODO(msteffen): Check if auth service is deployed. If not, print an
+			// error and exit
+			fmt.Println("(1) Please paste this link into a browser:\n" +
+				githubAuthLink + "\n" +
```

Comment:
I would add one more link break before and after the link to make it stand out more.